packages/jetbrains-plugin/src/test/kotlin/com/mongodb/jbplugin/fixtures/components/idea/IdeaFrame.kt

```diff
@@ -51,12 +53,13 @@ class IdeaFrame(
                 const file = LocalFileSystem.getInstance().findFileByPath(projectPath + '/' + path)
                 const openFileFunction = new Runnable({
                     run: function() {
-                        FileEditorManager.getInstance(project).openTextEditor(
-                            new OpenFileDescriptor(
-                                project,
-                                file
-                            ), true
-                        )
+                        const fileEditorManager = FileEditorManager.getInstance(project);
+                        const fileDescriptor = new OpenFileDescriptor(
+                            project,
+                            file
+                        );
+                        fileEditorManager.openTextEditor(fileDescriptor, true)
+                        fileEditorManager.navigateToTextEditor(fileDescriptor)
```

Comment:
Added this because the opened file was not getting navigated to.

Reply:
It makes sense, because the UI will open the editor when it's the only file open, but it might not do it with multiple files. Nice change!